styles/styles.css

```diff
@@ -0,0 +1,368 @@
+/* FONT STYLES */
+
+h1 {
+	font-family: "Lora", serif;
+	font-weight: 700;
+	color: #0e4769;
+}
+
+p {
+	font-family: "Open sans", sans-serif;
+}
+
+h3 {
+	font-family: "Lora", serif;
+	color: #0e4769;
+}
+
+.our-dogs-title {
+	margin: 3rem 0 3rem 0;
+}
+
+
+/* HEADER STYLING */
+
+header {
+	background-image: url("../images/header-image.jpg");
+	background-color: black;
+	background-size: cover;
+}
+
+.header-top {
+	margin: 0 1.5rem 4rem 1.5rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	;
+}
+
+.logo-bundle {
+	margin-top: 1.5rem;
+	margin-bottom: 1.5rem;
+}
+
+.shopping-cart-icon {
+	width: 1.5rem;
+}
+
+.logo-bundle p {
+	color: #2cc749;
+	font-family: "Lora", serif;
+	font-weight: 700;
+	font-style: italic;
+	font-size: 1.5rem;
+	margin: 0;
+	margin-right: 1rem;
+	padding: 0;
+}
+
+.header-logo {
+	width: 80%;
+}
+
+.shopping-cart p {
+	display: inline-block;
+	color: white;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+}
+
+.cart-button {
+	background-color: #2cc749;
+	border: none;
+	color: white;
+	padding: 1rem 2.5rem;
+	text-align: center;
+	text-decoration: none;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+	font-size: 1rem;
+	display: block;
+}
+
+
+/* HEADER STYLING: NAVIGATION */
+
+.header-nav {
+	background: #E4E4E4;
+	list-style: none;
+	padding: .5rem 0;
+}
+
+.header-nav li {
+	display: inline-block;
+	margin: 1rem 0;
+}
+
+.header-nav a {
+	text-decoration: none;
+	color: #253979;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+	font-size: 1rem;
+	padding: 1rem 2rem;
+}
+
+.header-nav a:hover {
+	background-color: #253979;
+	color: white;
+}
+
+
+/* FOOTER STYLING */
+
+footer {
+	background-color: #253979;
+}
+
+.footer-elements {
+	padding: 4rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.footer-logo {
+	width: 45%;
+}
+
+.footer-nav {
+	padding: 0;
+}
+
+.footer-nav li {
+	font-family: "Open Sans", sans-serif;
+	color: white;
+	display: inline-block;
+	margin: 1rem;
+}
+
+.footer-nav a {
+	text-decoration: none;
+	color: white;
+}
+
+.footer-elements p {
+	font-family: "Open sans", sans-serif;
+	font-size: 1rem;
+	width: 60%;
+	color: white;
+	margin-bottom: 0;
+	text-align: center;
+}
+
+small {
```

Comment:
this element name doesn't quite reflect what it does. I'd give it a class that's descriptive of what it is,  something like color-white